subsys/net/lib/aws_iot/src/aws_iot.c

```diff
@@ -508,7 +510,8 @@ static int topic_subscribe(void)
 		}
 	}
 
-	return err;
+	if (err < 0) return err;
```

Comment:
Checkpatch will probably want yoou to write this 
```
if (err < 0) {
        return err;
}
```

Reply:
Yes, fixed in the latest push.